docs/Maintainer-Guidelines.md

```diff
@@ -104,3 +104,14 @@ Whitespace corrections (to Ruby standard etc.) are allowed (in fact this
 is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
+
+## Communication
+Maintainers have a variety of ways to communicate with each other:
+
+- Homebrew's public repositories on GitHub
+- Homebrew's group communications between all maintainers on private channels (e.g GitHub/Slack/Discourse)
+- Homebrew's direct 1:1 messages between two maintainers on private channels (e.g. iMessage/Slack/Discourse/IRC/carrier pigeon)
```

Comment:
this wording leaves 3-way direct messaging ambiguous

Reply:
@ilovezfs Have made unambiguous by clarifying the above to be more than two